pages/game_page.py

```diff
@@ -1,3 +1,33 @@
 import streamlit as st
+import random
 
-st.title("GAME PAGE")
\ No newline at end of file
+st.title("Build a Scientist")
+
+game_words = ['Genetic','Histology','Sequence','Clinical Trial','Mutation','Genome','Oncology']
+
+def game_word_generator(word): 
+    st.write(word)
+    st.write(f"`{' '.join(['_'] * len(word))}`")
```

Comment:
I think we need to handle spaces differently here (or remove "Clinical Trial" as a game word option). Below, I add a space for spaces in the world instead of an underscore
```suggestion
    st.write(f"`{' '.join(['_ ' for char in word if char != ' ' else char])
```